pygmt/clib/loading.py

```diff
@@ -84,6 +78,37 @@ def clib_name(os_name):
     return libname
 
 
+def clib_full_names(env=None):
+    """
+    Return the full path of GMT's shared library for the current OS.
+
+    Parameters
+    ----------
+    env : dict or None
+        A dictionary containing the environment variables. If ``None``, will
+        default to ``os.environ``.
+
+    Returns
+    -------
+    lib_fullnames: list of str
+        List of possible full names of GMT's shared library.
+
+    """
+    if env is None:
+        env = os.environ
+    libnames = clib_name(os_name=sys.platform)
+    libpath = env.get("GMT_LIBRARY_PATH", "")
```

Comment:
I always found this section hard to follow, maybe add some examples here?
```suggestion
    libnames = clib_name(os_name=sys.platform)  # e.g. libgmt.so, libgmt.dylib, gmt.dll
    libpath = env.get("GMT_LIBRARY_PATH", "")  # e.g. $HOME/miniconda/envs/pygmt/lib
```